content/kubernetes/release-notes/k8s-6-2-18-41.md

```diff
@@ -0,0 +1,116 @@
+---
+Title: Redis Enterprise for Kubernetes release notes 6.2.18-41 (Dec 2022)
+linktitle: 6.2.18-41 (Dec 2022)
+description: This is a maintenance release for 6.2.18 and includes bug fixes.
+weight: 60
+alwaysopen: false
+categories: ["Platforms"]
+aliases: [
+    /kubernetes/release-notes/k8s-redis-on-flash.md,
+    /kubernetes/release-notes/k8s-redis-on-flash/, 
+    /kubernetes/release-notes/k8s-6-2-18-41.md,
+    /kubernetes/release-notes/k8s-6-2-18-41/,   ]
+---
+## Overview
+
+The Redis Enterprise K8s 6.2.18-41 is a maintenance release which supports the Redis Enterprise Software release 6.2.18 and contains bug fixes.
+
+The key bug fixes and known limitations are described below.
+
+## Images
+
+This release includes the following container images:
+
+* **Redis Enterprise**: `redislabs/redis:6.2.18-65` or  `redislabs/redis:6.2.18-65.rhel8-openshift` (or `redislabs/redis:6.2.18-65.rhel7-openshift` if upgrading from RHEL 7)
+* **Operator**: `redislabs/operator:6.2.18-41`
+* **Services Rigger**: `redislabs/k8s-controller:6.2.18-41` or `redislabs/services-manager:6.2.18-40` (on the Red Hat registry)
+
+## Bug fixes
+
+* Fixed issues with corrupt image sha (OLM/OpenShift)(RED-88863)
+* Fixed Active-Active upgrade issues wth NGINX (RED-88882)
+* Upgraded to Redis Software 6.2.18-65 (RED-88985)
+* Fixed upgrade issue with OLM (RED-89170)
+* Removed non-core Python dependencies from the log collector (RED-90108)
+* Fixed issues from running log collector on OLM (RED-90129
+
+## Compatibility notes
+
+Below is a table showing supported distributions at the time of this release. See [Supported Kubernetes distributions]({{< relref "/kubernetes/reference/supported_k8s_distributions.md" >}}) for the current list of supported distributions.
+
+| **Kubernetes version**  | 1.21       | 1.22       | 1.23       | 1.24       | 1.25       |
+|:------------------------|:----------:|:----------:|:----------:|:----------:|:----------:|
+| Community Kubernetes    |            | supported  | supported  | supported  | supported  |
+| Amazon EKS              | deprecated | supported  | supported  |            |            |
+| Azure AKS               |            | supported  | supported  | supported  |            |
+| Google GKE              | deprecated | supported  | supported  | supported  | supported* |
+| Rancher 2.6             | supported  | supported  | supported  | supported* |            |
+| VMware TKG 1.6          |            | supported* | supported* |            |            |
+| **OpenShift version**   | **4.8**    | **4.9**    | **4.10**   | **4.11**   |            |
+|                         |            | deprecated | supported  | supported* |            |
+| **VMware TKGI version** | **1.12**   | **1.13**   | **1.14**   | **1.15**   |            |
+|                         |            | supported  | supported  | supported* |            |
+
+\* Support added in this release
```

Comment:
I believe w should have the same table as in 6.2.18-3, but without any `*`'s since no platforms were added for the maint release.